docs/operation/volume_management/volume_creation.rst

```diff
@@ -0,0 +1,95 @@
+Creating Volumes
+================
+
+Create **Volume** objects to use persistent storage in a MetalK8s cluster.
+
+Requirements
+------------
+
+:doc:`StorageClass Objects
+</operation/volume_management/storageclass_creation>` are registered in the
+cluster.
+
+Using the CLI
+-------------
+
+#. Create a **Volume** manifest.
+
+   You can define a new volume using the following template:
+
+   .. code-block:: yaml
+
+       apiVersion: storage.metalk8s.scality.com/v1alpha1
+       kind: Volume
+       metadata:
+         name: <volume_name>
+       spec:
+         nodeName: <node_name>
+         storageClassName: <storageclass_name>
+         rawBlockDevice:
+           devicePath: <device_path>
+
+   Set the following fields:
+
+   - **name**: the name of your volume, must be unique.
+   - **nodeName**: the name of the node where the volume will be located.
+   - **storageClassName**: the name of the **StorageClass** to use.
+   - **devicePath**: path to the block device (for example, ``/dev/sda1``).
+
+#. Create the volume.
+
+   .. code-block:: shell
+
+      root@bootstrap $ kubectl apply -f volume.yml
+
+
+To check if the volume is successfully created, run the following command.
+
+   .. code-block:: shell
+
+       root@bootstrap $ kubectl get volume <volume_name>
+       NAME             NODE        STORAGECLASS
+       <volume_name>   bootstrap   metalk8s-demo-storageclass
+
+Using the UI
+------------
+
+#. Click **Nodes** on the left menu.
+
+    .. image:: /operation/volume_management/img/node_list.png
+
+#. Select the node on which to create the volume.
```

Comment:
You may violate the Prime Directive and end the sentence with a preposition here, if the spirit moves you.